constantTimeCompare.js

```diff
@@ -0,0 +1,46 @@
+  /**
+   * constantTimeCompare(x, y)
+   *
+   *  Created by Eric Elliott for the book,
+   *  Programming JavaScript Applications" (O'Reilly)
+   *
+   * MIT license http://opensource.org/licenses/MIT
+   */
+  
+// Regardless of the string size the number of character iterations/comparisons
+// ought to be equal to or higher than the maximum string size.
+var MAX_KEY_CHARS = 1024;
```

Comment:
There is no maximum string size in JavaScript, and the maximum hash size should increase over time. Where did you get this value?